gengokit/httptransport/templates/server.go

```diff
@@ -173,9 +180,12 @@ func (h httpError) Headers() http.Header {
 // EncodeHTTPGenericResponse is a transport/http.EncodeResponseFunc that encodes
 // the response as JSON to the response writer. Primarily useful in a server.
 func EncodeHTTPGenericResponse(_ context.Context, w http.ResponseWriter, response interface{}) error {
-	encoder := json.NewEncoder(w)
-	encoder.SetEscapeHTML(false)
```

Comment:
I can't see a way to disable the HTML escaping (`jsonpb` internally uses `encoding/json`, but doesn't currently have a way to call `SetEscapeHTML`), so I've filed an issue.